keps/sig-api-machinery/1965-kube-apiserver-identity/README.md

```diff
@@ -208,7 +204,7 @@ Alpha should provide basic functionality covered with tests described above.
 
 ###### Does enabling the feature change any default behavior?
 
-A namespace `kube-apiserver-lease` will be created to store kube-apiserver identity Leases.
+kube-apiserver will store identity Leases in the `kube-system` namespace.
 Old leases will be actively garbage collected by kube-controller-manager.
```

Comment:
this also needs updating since GC isn't done by kube-controller-manager any more

Reply:
fixed, thanks